src/System.Xml.XmlDocument/tests/RandomTests.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.IO;
+using Xunit;
+
+namespace System.Xml.Tests
+{
+    public class RandomTests
```

Comment:
Is there some name more descriptive we could use? Or an existing file this could go in?